linkedin_jobs_scraper/strategies/authenticated_strategy.py

```diff
@@ -33,7 +33,7 @@ class Selectors(NamedTuple):
     detailsPanel = '.jobs-search__job-details--container'
     detailsTop = '.jobs-details-top-card'
     details = '.jobs-details__main-content'
-    insights = '[class=jobs-unified-top-card__job-insight]'  # only one class
+    insights = '[class="mt5 mb2"]'  # only one class
```

Comment:
@gvteja99 I'm not sure if this is desired behavior, but this selector returns all insights combined in a list of length one.

I found the following selector returns each insight as a separate item within the list.

```suggestion
    insights = '[class="mt5 mb2"] > ul > li'  # only one class
```